packages/@ember/-internals/metal/tests/cached/get_test.js

```diff
@@ -0,0 +1,100 @@
+import { AbstractTestCase, moduleFor } from 'internal-test-helpers';
+import { cached, tracked } from '../..';
+
+moduleFor(
+  '@cached decorator: get',
+  class extends AbstractTestCase {
+    '@test it works'() {
+      let assert = this.assert;
+
+      class Person {
+        @tracked firstName = 'Jen';
+        @tracked lastName = 'Weber';
+
+        @cached
+        get fullName() {
+          let fullName = `${this.firstName} ${this.lastName}`;
+
+          assert.step(fullName);
+          return fullName;
+        }
+      }
+
+      let person = new Person();
+
+      assert.verifySteps([], 'getter is not called after class initialization');
+
+      assert.strictEqual(person.fullName, 'Jen Weber');
+      assert.verifySteps(['Jen Weber'], 'getter was called after property access');
+
+      assert.strictEqual(person.fullName, 'Jen Weber');
+      assert.verifySteps([], 'getter was not called again after repeated property access');
+
+      person.firstName = 'Kenneth';
+      assert.verifySteps([], 'changing a property does not trigger an eager re-computation');
+
+      assert.strictEqual(person.fullName, 'Kenneth Weber');
+      assert.verifySteps(['Kenneth Weber'], 'accessing the property triggers a re-computation');
+
+      assert.strictEqual(person.fullName, 'Kenneth Weber');
+      assert.verifySteps([], 'getter was not called again after repeated property access');
+
+      person.lastName = 'Larsen';
+      assert.verifySteps([], 'changing a property does not trigger an eager re-computation');
+
+      assert.strictEqual(person.fullName, 'Kenneth Larsen');
+      assert.verifySteps(['Kenneth Larsen'], 'accessing the property triggers a re-computation');
+    }
+
+    // https://github.com/ember-polyfills/ember-cached-decorator-polyfill/issues/7
```

Comment:
lets kill this comment (it doesn't really provide much more context than the test itself already does)

Reply:
done